### DISCUSSION TITLE
Is there any solution to validate a draft4 schema in the 2.2.0 release?

### DISCUSSION THREAD

**Discussion by willhuy** [pboettch/json-schema-validator#302] [Q&A] [answered]
Hello!

I'm intending to use this validator to validate against a draft4 schema but the validator only support draft7, is there any possible way I can get it to work with draft4 schema? 

Thank you!

**Comment by pboettch** [comment, marked as answer]
Draft4 and draft7 are close. Actually with some exceptions draft7 is backward compatible with draft4.

**Reply by willhuy** [reply]
I was able to convert the schema from draft4 to draft7 without problems, seems like a very simple problem but I didn't know who to figure out at first. Sorry for wasting the time ...
